lib/stat_tracker.rb

```diff
@@ -1,41 +1,45 @@
 require 'csv'
+require_relative 'game'
+require_relative 'team'
+require_relative 'game_teams'
+require_relative 'collection'
+require_relative 'game_collection'
+require_relative 'team_collection'
+require_relative 'game_teams_collection'
 
 class StatTracker
   attr_reader :games_collection, :teams_collection, :game_teams_collection
 
   def self.from_csv(locations)
-    game_file = locations[:games]
-    team_file = locations[:teams]
-    game_team_file = locations[:game_teams]
+    games = locations[:games]
+    teams = locations[:teams]
+    game_teams = locations[:game_teams]
 
-    StatTracker.new(game_file, team_file, game_team_file)
+    StatTracker.new(games, teams, game_teams)
   end
 
-  def initialize(game_file, team_file)
-    @games_collection = GameCollection.new(game_file)
-    @teams_collection = TeamCollection.new(team_file)
-    @game_teams_collection = GameTeamCollection.new(game_team_file)
+  def initialize(games, teams, game_teams)
+    @games_collection = GameCollection.new(games)
+    @teams_collection = TeamCollection.new(teams)
+    @game_teams_collection = GameTeamsCollection.new(game_teams)
   end
```

Comment:
Just want to make sure we actually add an initialization for seasons here so we can actually use it!